metricbeat/module/dropwizard/collector/collector.go

```diff
@@ -0,0 +1,96 @@
+package collector
```

Comment:
dropwizard? Is that here intentional? Sounds like an other potential module for the future :-)